omega/omega-transport/src/main/java/io/servicecomb/saga/omega/transport/TransactionClientHttpRequestInterceptor.java

```diff
@@ -0,0 +1,27 @@
+package io.servicecomb.saga.omega.transport;
+
+import java.io.IOException;
+
+import org.springframework.http.HttpRequest;
+import org.springframework.http.client.ClientHttpRequestExecution;
+import org.springframework.http.client.ClientHttpRequestInterceptor;
+import org.springframework.http.client.ClientHttpResponse;
+
+import io.servicecomb.saga.core.IdGenerator;
+
+public class TransactionClientHttpRequestInterceptor implements ClientHttpRequestInterceptor {
+
+  public static String TRANSACTION_ID_KEY = "X-Transaction-Id";
+
+  private IdGenerator<String> randomIdGenerator = new UniqueIdGenerator();
```

Comment:
please make it final. we may provide other implementation of `IdGenerator`. how about passing it in from constructor?